### DISCUSSION TITLE
Issue lacks the functionality to convert to Discussion

### DISCUSSION THREAD

**Discussion by erikr** [community/community#3088] [Discussions] [answered]
**Context:**
We recently enabled Discussions in [a repo](https://github.com/aguirre-lab/ml4c3) and look forward to using this new feature set.

**Current problem(s):**
1. Cannot convert [this issue](https://github.com/aguirre-lab/ml4c3/issues/1041) (or any issues in our repo) into a Discussion.
2. Cannot bulk convert issues based on labels on `github.com/org/repo/labels` page.

**Steps to Reproduce:**
1. Have sufficient permissions in our repo to convert an issue to a Discussion (I presume you need at least Triage).
2. Attempt to convert issue to Discussion, but fail because there is no button to do so.

**Expected behavior:**
1. Button exists in issues to convert to Discussions, which I'd expect per @becca https://github.com/github/feedback/discussions/2863#discussioncomment-171129.
2. Button exists to bulk convert issues to discussions by label.

**Comment by arkypita** [comment]
Same here. It was working since few day ago but today I no longer find "convert to discussion" button in my issue page.

**Reply by ninrod** [reply]
same problem here as @arkypita

**Comment by AsamK** [comment, marked as answer]
looks like it will be readded after some bugs are fixed:
https://github.com/github/feedback/discussions/2924#discussioncomment-254592

**Reply by becca** [reply]
Correct thank you for cross-referencing https://github.com/github/feedback/discussions/2924#discussioncomment-254592 @AsamK 🙇
